hydra_client/click.py

```diff
@@ -0,0 +1,157 @@
+""" This module contains helper functions and utilities for generating plugin.xml files from a click CLI.
+"""
+import click
+from pathlib import Path
+from xml.etree import ElementTree as ET
+from xml.dom.minidom import parseString
+import hydra_base
+import os
+
+
+# Special argument names (keys) to be used as click options mapped
+# to types defined in Hydra/HWI (values)
+ARGTYPES = {
```

Comment:
can we name this to something more meaningful, like HYDRA_ARGTYPES or something?